### PR TITLE
Enhance documents about the data report and query protocols.

### PR DESCRIPTION
**Pull request by wu-sheng**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: docs/en/protocols/README.md [hunk 1]
```diff
@@ -1,51 +1,46 @@
...
 
 [Cross Process Propagation Headers Protocol v3](Skywalking-Cross-Process-Propagation-Headers-Protocol-v3.md) has been the new protocol for in-wire context propagation since the version 8.0.0 release.
 
 [Cross Process Correlation Headers Protocol v1](Skywalking-Cross-Process-Correlation-Headers-Protocol-v1.md) is a new in-wire context propagation protocol which is additional and optional. 
-Please read SkyWalking language agents documentation to see whether it is supported. 
-This protocol defines the data format of transporting custom data with `Cross Process Propagation Headers Protocol`.
-It has been supported by the SkyWalking javaagent since 8.0.0, 
+Please read SkyWalking language agents documentation to see whether it is supported.
+
+- **Trace Data Protocol** is an out-of-wire data format. Agent/SDK uses this to send traces to SSkyWalking OAP server.
```
Comment (dmsolr) [inline docs/en/protocols/README.md:19]: - **Trace Data Protocol** is an out-of-wire data format. Agent/SDK uses this to send traces to SkyWalking OAP server.

Comment (wu-sheng) [reply docs/en/protocols/README.md]: Fixed.

File: docs/en/protocols/query-protocol.md [hunk 2]
```diff
@@ -127,12 +127,51 @@ extend type Query {
...
+
+    # Test the logs and get the results of the LAL output.
+    test(requests: LogTestRequest!): LogTestResponse!
+}
+```
+
+Log implementations have a little different with different database options. Search engine(s), e.g. ElasticSearch and OpenSearch, could support
+full log text fuzzy query. Others would not support considering performance impact and end user experience.
+
+`test` API is provides for the debugger tool of native LAL parsing.
```
Comment (kezhenxu94) [inline docs/en/protocols/query-protocol.md:145]:
```suggestion
`test` API is provided for the debugger tool of native LAL parsing. 
```

File: docs/en/protocols/query-protocol.md [hunk 3]
```diff
@@ -127,12 +127,51 @@ extend type Query {
...
+    # Return true if the current storage implementation supports fuzzy query for logs.
+    supportQueryLogsByKeywords: Boolean!
+    queryLogs(condition: LogQueryCondition): Logs
+
+    # Test the logs and get the results of the LAL output.
+    test(requests: LogTestRequest!): LogTestResponse!
+}
+```
+
+Log implementations have a little different with different database options. Search engine(s), e.g. ElasticSearch and OpenSearch, could support
```
Comment (kezhenxu94) [inline docs/en/protocols/query-protocol.md:142]:
```suggestion
Log implementations have a little differences with different database options. Search engine(s), e.g. ElasticSearch and OpenSearch, could support
```

**Review by dmsolr — APPROVED** [review]
LGTM

**Review by kezhenxu94 — COMMENTED** [review]
As it's all about doc updates. FYI @wingwong-knh in case you are available to take a look.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**